src/components/organisms/TimingEventModal/TimingEventModal.scss

```diff
@@ -40,4 +40,25 @@
   .ButtonNG {
     max-width: none;
   }
+
+  .AdminDateTime {
+    margin-top: 0;
+    margin-bottom: 0.8rem;
+  }
+
+  .AdminDateTime__input-wrapper {
+    padding: 0;
+    input {
+      border: 0.1px solid var(--accent--over-20a);
+    }
+  }
+
+  // This is to combat the use of a h2 tag in the wrong place. Longer term
+  // we probably want to NOT use AdminSection inside this form.
+  .AdminSection {
+    h2 {
+      font-weight: $font-weight--400;
+      font-size: font-size--root();
+    }
+  }
```

Comment:
Maybe not the correct time to use `AdminDateTime` component inside this form if the more specific component (form) can't be adjusted accordingly, but the more general one (AdminDateTime) must accommodate its specificities.